pages/5_Compiling.py

```diff
@@ -1,15 +1,33 @@
 import streamlit as st
 
+from tflm_hello_world.training import model_training
+
 st.set_page_config(
     page_title = 'Compiling',
     page_icon = '✅',
     layout = 'wide'
 )
 
+if "compile" not in st.session_state:
+    st.session_state["compile"]="not done"
 
 st.title('Compiling')
 st.header('Compiling Hello World TinyML model')
 st.subheader('with TFLm tool')
 st.markdown('- Choose compilation options')
-st.markdown('- Compile')
+st.markdown('- Compile into a deployable Docker image')
```

Comment:
Why docker mentioned here?

Reply:
Oh, that was a mistake. That was from when i wasn't sure what exactly was the goal of the compiling section

Reply:
Since the acceptance tests said "WebApp generates a runnable Docker binary, which is supposed to predict human detection(?)"

Reply:
But thats the "installing" sections job right?

Reply:
yes